app/views/books/show.html.erb

```diff
@@ -1,10 +1,26 @@
 <div>
   <h1>Title</h1>
-  <%= book.title %>
-  <h1>ISBN</h1>
+  <h1><%= book.title %></h1>
+  <h3>ISBN</h3>
   <%= book.isbn %>
-  <h1>Year of Published</h1>
+  <h3>Year of Published</h3>
   <%= book.year %>
-  <h1>Author</h1>
+  <h3>Author</h3>
   <%= "#{book.author.first_name}  #{book.author.last_name}"%>
+
+  <h1>Comment List</h1>
+  <% book.comments.each do |comment| %>
```

Comment:
Check if you have the N+1 queries problem here. I think you are not eager loading the comments when you load the book.